src/istgt.c

```diff
@@ -2334,7 +2345,6 @@ istgt_acceptor(ISTGT_Ptr istgt)
 	event.events = EPOLLIN;
 	rc = epoll_ctl(epfd, EPOLL_CTL_ADD, istgt->sig_pipe[0], &event);
 	if (rc == -1) {
-		MTX_UNLOCK(&istgt->mutex);
```

Comment:
instead of removing UNLOCK code here, extend the lock i.e., remove UNLOCK at line#2315 of new file, and extend the lock to below this code.

Reply:
ok